doc/whats_new/v0.23.rst

```diff
@@ -339,6 +339,11 @@ Changelog
   computing statistics when calling `partial_fit` on sparse inputs.
   :pr:`16466` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| Fix a bug in :class:`preprocessing.Normalizer` which was not taking the
```

Comment:
Let's try make this a bit more relevant for the user: please mention "with norm='max'".